packages/google_maps_flutter/google_maps_flutter_web/lib/src/convert.dart

```diff
@@ -62,9 +62,20 @@ gmaps.MapOptions _configurationAndStyleToGmapsOptions(
 
   if (configuration.scrollGesturesEnabled == false ||
       configuration.zoomGesturesEnabled == false) {
-    options.gestureHandling = 'none';
+    window.console.warn(
+      '`scrollGesturesEnabled` and `zoomGesturesEnabled` are now ignored on the web.'
+      ' Use `webGestureHandling: WebGestureHandling.none` instead.',
+    );
+  }
+
+  if (configuration.webGestureHandling != null) {
+    options.gestureHandling = configuration.webGestureHandling!.name;
   } else {
```

Comment:
```suggestion
  } else if (configuration.scrollGesturesEnabled == false ||
      configuration.zoomGesturesEnabled == false) {
      // Old behavior
      options.gestureHandling = 'none';
  } else {
    options.gestureHandling = WebGestureHandling.auto.name;
  }
```

/cc @stuartmorgan is this what you mean? (and remove the warn?)

Reply:
Yes, exactly (unless there's a reason that won't work, it seems like it's the most backwards-compatible behavior).